ironfish/src/rpc/routes/multisig/types.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
I assume these types are gonna get used in another RPC? Seems good to me!

Reply:
Yep